index.js

```diff
@@ -53,7 +66,16 @@ function feedMetaTags({ title, description, link, copyright, generator, docs, op
       { lastBuildDate: format(now, 'ddd, DD MMM YYYY HH:mm:ss ZZ') }, // Date format must be RFC 822 compliant
       { docs: docs || 'http://blogs.law.harvard.edu/tech/rss' },
       { copyright: copyright || now.getFullYear() },
-      { generator: generator || 'Feed delivered by Clay' }
+      { generator: generator || 'Feed delivered by Clay' },
+      { language: 'en-us' },
+      { 'itunes:subtitle': description },
+      { 'itunes:author': title },
+      { 'itunes:summary': description },
+      { 'itunes:type': 'episodic' },
+      { 'itunes:owner': contactInfo(title, 'neil.janowitz@vulture.com') },
```

Comment:
the `owner`, `image`, `category`, and `explicit` fields should be generated from data in your Clay instance rather than hardcoded. (For NYMag, I suspect most of that data can come from the `article` component, including using `Not Safe For Ads` for `explicit`)